web/src/components/SearchAndFilter.tsx

```
@@ -0,0 +1,129 @@
+        </div>
+        {activeFiltersCount > 0 && (
+          <Button variant="outline" size="sm" onClick={clearFilters}>
+            Clear All
+          </Button>
+        )}
+      </div>
+
+      <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-4 gap-4">
+        {/* Search */}
+        <div className="space-y-2">
+          <label className="text-sm font-medium">Search</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)